### PR TITLE
makes genome file creation more specific

### LINKED ISSUE #2002: bedtools coverage cannot handle contig names containing ":"

### PR DESCRIPTION
**Pull request by bwlang**
fixes #2002

### CODE REVIEW

**Comment by bwlang** [comment]
@bernt-matthias  i think you're also adding -sorted support to intersect... this technique might be of interest to you

File: tools/bedtools/coverageBed.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
         #if str($reduce_or_iterate.reduce_or_iterate_selector) == 'iterate':
             -b '$reduce_or_iterate.inputB'
             #if $reduce_or_iterate.inputB.is_of_type('bam'):
-                -sorted  -g <(samtools view -H $reduce_or_iterate.inputB | tr ':' '\t' | grep SN | cut -f 3,5)
+                -sorted  -g <(samtools view -H $reduce_or_iterate.inputB | grep '^@SQ' |  sed 's/\tSN:/\tSN\t/' | sed 's/\tLN:/\tLN\t/' cut -f 3,5)
```
Comment (abretaud) [inline tools/bedtools/coverageBed.xml]: I think a | is missing before cut?
(btw, is it complicated to have a test for this bug?)

Comment (bwlang) [reply tools/bedtools/coverageBed.xml]: Ack! yes - a test is needed here - i thought there already was one.

**Comment by abretaud** [comment]
Thanks!

**Pull request merged**